extensions/wikia/DesignSystem/ds.built.js

```diff
@@ -0,0 +1,768 @@
+					return memo;
+				};
+			},
+			isOldIE = memoize(function() {
+				// Test for IE <= 9 as proposed by Browserhacks
+				// @see http://browserhacks.com/#hack-e71d8692f65334173fee715c222cb805
+				// Tests for existence of standard globals is to allow style-loader
+				// to operate correctly into non-standard environments
+				// @see https://github.com/webpack-contrib/style-loader/issues/177
+				return window && document && document.all && !window.atob;
+			}),
+			getElement = (function(fn) {
+				var memo = {};
+				return function(selector) {
+					if (typeof memo[selector] === "undefined") {
+						memo[selector] = fn.call(this, selector);
+					}
+					return memo[selector]
+				};
+			})(function (styleTarget) {
+				return document.querySelector(styleTarget)
+			}),
+			singletonElement = null,
+			singletonCounter = 0,
+			styleElementsInsertedAtTop = [],
+			fixUrls = __webpack_require__(8);
+
+		module.exports = function(list, options) {
+			if(typeof DEBUG !== "undefined" && DEBUG) {
+				if(typeof document !== "object") throw new Error("The style-loader cannot be used in a non-browser environment");
+			}
+
+			options = options || {};
+			options.attrs = typeof options.attrs === "object" ? options.attrs : {};
+
+			// Force single-tag solution on IE6-9, which has a hard limit on the # of <style>
+			// tags it will allow on a page
+			if (typeof options.singleton === "undefined") options.singleton = isOldIE();
+
+			// By default, add <style> tags to the <head> element
+			if (typeof options.insertInto === "undefined") options.insertInto = "head";
+
+			// By default, add <style> tags to the bottom of the target
+			if (typeof options.insertAt === "undefined") options.insertAt = "bottom";
+
+			var styles = listToStyles(list);
+			addStylesToDom(styles, options);
+
+			return function update(newList) {
+				var mayRemove = [];
+				for(var i = 0; i < styles.length; i++) {
+					var item = styles[i];
+					var domStyle = stylesInDom[item.id];
+					domStyle.refs--;
+					mayRemove.push(domStyle);
+				}
+				if(newList) {
+					var newStyles = listToStyles(newList);
+					addStylesToDom(newStyles, options);
+				}
+				for(var i = 0; i < mayRemove.length; i++) {
+					var domStyle = mayRemove[i];
+					if(domStyle.refs === 0) {
+						for(var j = 0; j < domStyle.parts.length; j++)
+							domStyle.parts[j]();
+						delete stylesInDom[domStyle.id];
+					}
+				}
+			};
+		};
+
+		function addStylesToDom(styles, options) {
+			for(var i = 0; i < styles.length; i++) {
+				var item = styles[i];
+				var domStyle = stylesInDom[item.id];
+				if(domStyle) {
+					domStyle.refs++;
+					for(var j = 0; j < domStyle.parts.length; j++) {
+						domStyle.parts[j](item.parts[j]);
+					}
+					for(; j < item.parts.length; j++) {
+						domStyle.parts.push(addStyle(item.parts[j], options));
+					}
+				} else {
+					var parts = [];
+					for(var j = 0; j < item.parts.length; j++) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Rename "j" as this name is already used in declaration at line 433. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2814)